tvaultuiv2/src/services/helper-function.js

```diff
@@ -40,19 +40,29 @@ export function findElementById(arr, id, nestingKey) {
     'Not found'
   );
 }
+export const findItemAndRemove = (arr, key, id) => {
+  if (arr?.length === 0) return;
+  const tempArr = [...arr];
+  const indexofItem = tempArr[0][key].findIndex((item) => item.id === id);
```

Comment:
```suggestion
  const indexofItem = tempArr[0][key].findIndex((item) => item.id === id);
```

null chack for tempArr[0]??